### PR TITLE
Create Lecture Halls

### PR DESCRIPTION
**Pull request by MatthiasReumann**
- Added create button to lectureHalls page
- Added page/template "create-lecture-halls" -> /admin/lectureHalls/new

### CODE REVIEW

**Comment by joschahenningsen** [comment]
Nice!

Your npm version might be a little outdated, seems like it changed the lockfile to version 1, I'm on npm 8.x.

I just merged #191, do you think merging these changes is manageable?

**Comment by alexanderstephan** [comment]
@joschahenningsen I think you meant https://github.com/joschahenningsen/TUM-Live/pull/186?

**Comment by joschahenningsen** [comment]
> @joschahenningsen I think you meant #186?

Ah yes, right, that one :D

**Comment by joschahenningsen** [comment]
This looks very good, thanks! 
I think after creating a lecture hall, there should either be a visual indicator that nothing went wrong, or a redirect to the lecture hall overview (or both)

File: api/lecture_halls.go [hunk 1]
```diff
@@ -368,6 +368,8 @@ func createLectureHall(c *gin.Context) {
 		CombIP: req.CombIP,
 		PresIP: req.PresIP,
 		CamIP:  req.CamIP,
+		CameraIP: req.CameraIP,
```
Comment (alexanderstephan) [inline api/lecture_halls.go:371]: `CamIP` vs. `CameraIP` is a bit confusing, maybe we should rename it in a later PR. It is fine for now though.

**Review by alexanderstephan — COMMENTED** [review]
Code changes look good to me! Everything works fine locally as well. Maybe we should add a `Done` button beneath the `Create` button in ` lectureHalls/new `, that loads the lecture hall overview again (I think it helps with usability as one probably wants to go back to the overview after adding a lecture hall).

**Comment by MatthiasReumann** [comment]
> Code changes look good to me! Everything works fine locally as well. Maybe we should add a `Done` button beneath the `Create` button in `lectureHalls/new`, that loads the lecture hall overview again (I think it helps with usability as one probably wants to go back to the overview after adding a lecture hall).

I guess, as Joscha suggested, a redirect to the overview page after successfully adding a lecture hall is a good idea. Forgot to change that last time. ^^

**Review by alexanderstephan — APPROVED** [review]
Ah, okay. Looks good to me now, redirect works as expected! CodeQL seems to be somehow broken right now, but that should not be related to this PR.

**Review by joschahenningsen — APPROVED** [review]
Looks great, thanks! 

running `git config core.fileMode false` should get rid of file mode changes in future commits :)

**Pull request merged**